src/dist/betaprime.rs

```diff
@@ -17,6 +16,8 @@ use std::sync::OnceLock;
 #[cfg(feature = "experimental")]
 use super::UnitPowerLaw;
 
+#[cfg(feature = "experimental")]
+use crate::traits::{ConjugatePrior, DataOrSuffStat, HasSuffStat};
```

Comment:
Why the feature flag?

Reply:
Compiler likes it, just not seeing why

Reply:
If experimental is disabled, this shows an unused import error.